docs/platforms/apple/common/index.mdx

```diff
@@ -85,9 +91,130 @@ If you prefer, you can also [set up the SDK manually](/platforms/apple/guides/io
 
 To capture all errors, initialize the SDK as soon as possible, such as in your `AppDelegate` `application:didFinishLaunchingWithOptions` method:
 
-<PlatformSection notSupported={["apple.tvos", "apple.watchos", "apple.visionos"]}>
+<PlatformSection supported={["apple.ios"]}>
+
+```swift {tabTitle:Swift} {"onboardingOptions": {"performance": "13-16", "profiling": "24-34", "session-replay": "17-22"}}
+import Sentry
+
+func application(_ application: UIApplication,
+    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+    SentrySDK.start { options in
+        options.dsn = "___PUBLIC_DSN___"
+        options.debug = true // Enabled debug when first installing is always helpful
+
+        // Adds IP for users.
+        // For more information, visit: https://docs.sentry.io/platforms/apple/data-management/data-collected/
+        options.sendDefaultPii = true
+
+        // Set tracesSampleRate to 1.0 to capture 100% of transactions for performance monitoring.
+        // We recommend adjusting this value in production.
+        options.tracesSampleRate = 1.0
+
+        // Record Session Replays for 100% of Errors and 10% of Sessions
+        options.sessionReplay.onErrorSampleRate = 1.0
+        options.sessionReplay.sessionSampleRate = 0.1
+        // We recommend the ~5x more performant experimental view renderer
+        options.sessionReplay.enableExperimentalViewRenderer = true
```

Comment:
we need to decide if we want to point this out here. IMO we should, or we postpone this PR for iOS (and RN, see below)